rpa-home/modules/ROOT/pages/glossary.adoc

```diff
@@ -65,6 +81,14 @@ See the https://www.mulesoft.com/prod-subscription-plans[MuleSoft Product Subscr
 
 ''''''
 
+=== IDP 
+
+Intelligent Document Processing is a technology solution that implements AI to read documents, extract data, and organize information from different document sources.
+
+=== Invokable Run Configuration 
+
+An automated process that you can upload to exchange and then invoke from other services as Composer. 
```

Comment:
The configuration is created in RPA Manager and with saving it, it will automatically uploading an invokable API in the exchange of the anypoint org.
So "automated process" is not the right term here. For more info see https://docs.mulesoft.com/rpa-home/invoke-rpa-services and https://docs.mulesoft.com/rpa-manager/processautomation-deploy#publish-automation-exchange